app/qml/map/MMMapController.qml

```diff
@@ -922,6 +942,43 @@ Item {
     }
   }
 
+
+  Loader {
+    id: multiEditLoader
+
+    anchors.fill: mapCanvas
+
+    // active: true
+    active: root.state === "select"
```

Comment:
Could we use some other term than `select` for this functionality within the code? It somehow does not immediately hit me that this is the multi-select functionality we talk about. How about `mutliselection` or `multiselect`?

e.g. I get `onMultiSelect` instead of `onSelect`